src/lesson6/CreditCard.java

```diff
@@ -0,0 +1,32 @@
+package lesson6;
+
+public class CreditCard {
+    private final int accountNumber;
+    private int balance;
+
+    public CreditCard(int accountNumber, int balance) {
+        this.accountNumber = accountNumber;
+        this.balance = balance;
+    }
+
+
+    public void replenishmentCard(int sum) {
+        this.balance += sum;
+    }
+
+    public int withdrawCard(int sum) {
+        if (this.balance >= sum) {
+            this.balance -= sum;
+        } else {
+            System.out.printf("На счету %d недостаточно средств", this.accountNumber);
```

Comment:
I think, in else block we could return -1 for example, as a value of an errored operation